CSV.js

```diff
@@ -195,6 +196,16 @@ function getValue(item, field) {
 			}
 			value += escapeValue(notes.join(valueSeparator));
 		break;
+		case 'date':
+			if (item.date) {
+				var dateISO = ZU.strToISO(item.date);
```

Comment:
I don't know -- since you want to get to the else both if `normalizeDate` is false *and* if dateISO is empty (e.g. the date is just a word or so), this seems like the most elegant approach to me.